.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 100,
+  "singleQuote": true,
```

Comment:
We've had some issues on a project because of single quotes on SCSS files. Specifically while trying to target RTL where `div[dir="rtl"]` and `div[dir=rtl]` are valid syntax but `div[dir='rtl']` isn't.

```
    {
      "files": "*.scss",
      "options": {
        "singleQuote": false
      }
    }
```